themes-default/slim/views/config_search.mako

```diff
@@ -14,9 +14,12 @@ window.app = {};
 const startVue = () => {
     window.app = new Vue({
         el: '#vue-wrap',
+        store,
         metaInfo: {
             title: 'Config - Episode Search'
         },
+        computed: Object.assign(store.mapState([]), {
```

Comment:
You pull the config but don't assign anything. Missed it?
Now that `config` is available from the global mixin,
maybe you don't need to pull the config here or map the state at all?